tests/test_agents.py

```diff
@@ -1080,6 +1080,19 @@ async def test_search_pagination(agent_test_settings: Settings) -> None:
     ), "Second page should match second slice of all results"
 
 
+@pytest.mark.asyncio
+async def test_agent_empty_index(agent_test_settings: Settings):
```

Comment:
```suggestion
async def test_empty_index_without_index_rebuild(agent_test_settings: Settings):
```

Just being more explicit here